docs/timeseries/lombscargle.rst

```diff
@@ -621,16 +688,16 @@ In all of this, it is important to keep in mind a few caveats:
 For a detailed discussion of these caveats and others when computing and
 interpreting false alarm probabilities, please refer to [11]_.
 
-
 Periodogram Algorithms
 ======================
+
 The :class:`~astropy.timeseries.LombScargle` class makes available
 several complementary implementations of the Lomb-Scargle Periodogram,
 which can be selected using the ``method`` keyword of the Lomb-Scargle power.
 By design all methods will return the same results (some approximate),
 and each has its advantages and disadvantages.
 
-For example, to compute a periodogram using the fast chi-square method
+For example, to compute a periodogram using the Fast Chi-squared method
```

Comment:
I don't think this needs the capitalization

Reply:
It's capitalized on the arxiv: https://arxiv.org/abs/0901.1913 but is it common enough that people generally use it less formally?

Reply:
Interesting. I would have assumed it is the same  logic as "fast" in the fast Fourier transform, which is capitalized in the abbreviation but not in full text form, that is also along the line that`chi-squared` is usually kept lower case, or is written with the Greek letter

Reply:
Looks like Fast Fourier Transform is also all caps in that paper, so maybe they just like capitalizing!